source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -0,0 +1,2150 @@
+    margin: 0px 0px 20px 0px;
+    padding: 0;
+    font-size: 1.5rem;
+  }
+
+  .ca_non_favouriteRepo {
+    font-size: 2rem;
+    cursor: pointer;
+    margin-left: 2.5rem !important;
+    padding-right: .5rem;
+    cursor: pointer;
+    padding: .3rem;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-right



(lint/suspicious/noShorthandPropertyOverrides)